geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -8,6 +8,8 @@ Information
 -----------
 
 1. From now the basic auth is disabled by default.
```

Comment:
Sentence is not correct. Suggestion: 
"Basic authentication is disabled by default from this version onward."